packages/web/src/hooks/useTabs/useTabs.tsx

```diff
@@ -965,6 +968,7 @@ const useTabs = ({
   elements,
   didChangeTabsFrom,
   didTransitionCallback,
+  onTabClick: onTabClickCb,
```

Comment:
Needed to add this because `didChangeTabsFrom` doesn't execute the callback until the tab animation has finished.